README.md

```diff
@@ -276,11 +285,14 @@ after answering. See the example below.
 
 ### 2. Feedback questionnaire
 
-A feedback questionnaire is basically just like a graded questionnaire, but with
-the `feedback` option it uses the feedback category and CSS class by default.
+A feedback questionnaire is almost like a graded questionnaire. With
+the `feedback` option questionnaire uses the feedback category and CSS class by default.
 The options `chapter-feedback`, `weekly-feedback`, `appendix-feedback`, and
 `course-feedback` use a different CSS class (with the same name as the option).
-If points are not specified, they are set to zero.
+If points are not specified, they are set to zero. The `feedback` option can be set only to the last questionnaire of the RST file.
+
+The freetext questions are not expected to have the model solution, in essence the
```

Comment:
One line is very long.

"the model solution" should be "a model solution" here.